concepts/basics/about.md

```diff
@@ -0,0 +1,214 @@
+# About
+
+Julia is a dynamic, strongly-typed programming langauge.
+The programming styple is mainly functional, though with more flexibility than in languages such as Haskell.
+
+There is a strong and versatile type system, which will become important in later concepts. In practice, Julia will usually infer a suitable default from the context.
+
+Despite its emphasis on readability, making it appear as a scripting language like Python and Ruby, Julia code is compiled before running.
+The Just In Time (JIT) compiler produces highly optimized code for each function as it is enountered, leading to high runtime speed.
+
+Julia is not an object-oriented language, and there is no class hierarchy of the sort central to many other languages.
+
+Instead *(and please don't panic as you read this!)*, Julia relies on:
+- A *Type* hierarchy.
+- *Composable* functions (designed to combine easily, in versatile and powerful ways).
+- *Multiple dispatch*, meaning functions can be called with variable number and types of arguments.
+
+These rather confusing terms should become clearer as we progress through the syllabus.
+
+The [offical documentation][official-documentation] contains a lot of valuable information for the current release, though it can be a bit overwhelming for beginners.
```

Comment:
```suggestion
The [official documentation][official-documentation] contains a lot of valuable information for the current release, though it can be a bit overwhelming for beginners.
```